File: meldingsvarsler-oppgave/src/main/java/no/sonat/meldingsvarsler/abonnent/SMSAbonnent.java
package no.sonat.meldingsvarsler.abonnent;

public class SMSAbonnent {
    private final String navn;
    private final String telefonnummer;

    public SMSAbonnent(String navn, String telefonnummer) {
        this.navn = navn;
        this.telefonnummer = telefonnummer;
    }

    public String navn() {
        return navn;
    }

    public Boolean erAktiv() {
        return true;
    }

    public String telefonnummer() {
        return telefonnummer;
    }
}

File: meldingsvarsler-oppgave/App/SMSAbonnent.cs
namespace App
{
    public class SmsAbonnent
    {
        public SmsAbonnent(string navn, string telefonNummer)
        {
            Navn = navn;
            TelefonNummer = telefonNummer;
        }

        public string Navn { get; }

        public bool ErAktiv => true;

        public string TelefonNummer { get; }
    }
}